WcaOnRails/spec/factories/competitions.rb

```diff
@@ -44,5 +44,10 @@
       with_delegate
       isConfirmed true
     end
+
+    trait :confirmed_and_visible do
```

Comment:
I thought about creating a :visible trait which sets `showAtAll true` but if it the call to FactoryGirl only uses this trait the competition will be invalid because it needs a delegate. I could add `with_delegate` to the visible trait but then if you call `[:confirmed, :visible]` do you get 2 delegates? Or is it fine to just set showAtAll for  the visible trait and then if you want a valid competition you need to add either `with_delelgate` or `confirmed`?